### PR TITLE
Get single milestone

### PR DESCRIPTION
**Pull request by anamariamv**
Implement https://developer.github.com/v3/issues/milestones/#get-a-single-milestone

### CODE REVIEW

File: docs/docs/issue.md [hunk 1]
```diff
@@ -463,3 +464,29 @@ The `result` on the right is the corresponding [Milestone][milestone-scala]
 See [the API doc](https://developer.github.com/v3/issues/milestones/#create-a-milestone) for full reference.
 
 [milestone-scala]: https://github.com/47degrees/github4s/blob/master/github4s/src/main/scala/github4s/domain/Milestone.scala
+
+### Get a single milestone
+
+You can also get a single milestone of a repository through `getMilestone`; it takes as arguments:
+
+- `number`: The milestone number.
```
Comment (juanpedromoreno) [inline docs/docs/issue.md]: `number` is after `repo`, isn't it?

Comment (anamariamv) [reply docs/docs/issue.md]: Yes, 👍

File: docs/docs/issue.md [hunk 2]
```diff
@@ -463,3 +464,29 @@ The `result` on the right is the corresponding [Milestone][milestone-scala]
...
+- `owner`: name of the owner for which we want to retrieve the milestones.
+- `repo`: name of the repository for which we want to retrieve the milestones.
+- `header`: headers to include in the request, optional.
+
+ The milestone for owner `47deg` and repository `github4s`:
```
Comment (BenFradet) [inline docs/docs/issue.md]:
```suggestion
 To get milestone number 3254 for owner `47deg` and repository `github4s`:
```

File: github4s/src/test/scala/github4s/integration/GHIssuesSpec.scala [hunk 3]
```diff
@@ -273,4 +273,21 @@ trait GHIssuesSpec extends BaseIntegrationSpec {
     response.statusCode shouldBe notFoundStatusCode
   }
 
+  "GHIssues >> GetMilestone" should "return error for an invalid number" taggedAs Integration in {
```
Comment (BenFradet) [inline github4s/src/test/scala/github4s/integration/GHIssuesSpec.scala]: could we have a test that is succeeding?

Comment (anamariamv) [reply github4s/src/test/scala/github4s/integration/GHIssuesSpec.scala]: We don't have any milestones in this repository.  :(

Comment (BenFradet) [reply github4s/src/test/scala/github4s/integration/GHIssuesSpec.scala]: https://github.com/47degrees/github4s/milestone/1 😄

**Pull request merged**